pages/jobs/index.html.njk

```diff
@@ -10,29 +10,31 @@ nav-index: 2
   <h1>Jobs</h1>
   <p>If you have any questions please reach out to us at: <a href="mailto:talent-buildit@wipro.com">talent-buildit@wipro.com</a></p>
 </div>
-{% for jobLocation in jobLocations %}
+{% for country, cities in jobsByCountryAndCity | dictsort %}
+<h2>{{ country }}</h2>
+{% for city, jobs in cities | dictsort %}
+{% set jobLocation = jobs[0].location %}
+{% set studio = locationsByCity[city][0] %}
 <section>
-  <header typeof="schema:Place" resource="https://buildit.wiprodigital.com/thing/job-location/{{jobLocation.location.countryCode}}/{{jobLocation.location.citySlug}}">
-    <h2 property="schema:address" typeof="schema:PostalAddress">
+  <header typeof="schema:Place" resource="{{jobLocation.uri}}">
+    <h3 property="schema:address" typeof="schema:PostalAddress">
     <span property="schema:name">
-      <span property="schema:addressLocality">{{jobLocation.location.city}}</span>,
-      {{jobLocation.location.country}}
+      <span property="schema:addressLocality">
+      {% if studio %}
+        <a href="/{{ studio.path }}">{{ city }}</a>
+      {% else %}
+        {{ city }}
+      {% endif %}
+      </span>
     </span>
-      <meta property="schema:addressCountry" content="{{jobLocation.location.countryCode}}">
-    </h2>
+      <meta property="schema:addressCountry" content="{{jobLocation.countryCode}}">
+    </h3>
   </header>
   <ul class="grav-o-two-column grav-o-column-list">
-    {% for job in jobLocation.jobs %}
-    <li typeof="schema:JobPosting">
-      <a rel="schema:url" property="schema:title" href="{{job.url}}">{{job.title}}</a>
-      <meta property="schema:datePosted" content="{{job.datePosted}}">
-      <meta property="schema:employmentType" content="{{job.typeOfEmployment}}">
-      <meta property="schema:industry" content="{{job.industry}}">
-      <meta property="schema:description" content="For the full job description please visit: {{job.url}}">
-      <link property="schema:hiringOrganization" href="https://buildit.wiprodigital.com/thing/buildit-org">
-      <link property="schema:jobLocation" href="https://buildit.wiprodigital.com/thing/job-location/{{jobLocation.location.countryCode}}/{{jobLocation.location.citySlug}}">
-    </li>
+    {% for job in jobs %}
+      {% include 'partials/jobs/job-card.njk' %}
     {% endfor %}
   </ul>
 </section>
 {% endfor %}
+{% endfor %}
```

Comment:
With the country groupings I find it quite difficult to parse this page visually. Also, the fact that some of the locations have links to a detail page and other's do not is odd. I know I understand the difference - but i'd suspect it's seemingly random to a new user.

Reply:
I mocked up some alterations which I think make the page more readable:
![localhost_8080_jobs_](https://user-images.githubusercontent.com/46753034/60035012-bf032200-96a3-11e9-831e-d4133d4bc53a.png)

They include:
- Making `<h3>`s much smaller relative to their `<h2>` "parent"
- Adding the `<hr>` in between each country
- Making the default margin between the job list items smaller (1rem instead of 2rem in this case)
- On the Edinburgh entry: Removing the link from the `<h2>` and adding in a `<p>` with a short piece of text and link to the location page as per @james-nash's suggestion. I would do this to all of them but was just playing in the inspector :)

Reply:
Yes. I'll be honest, I didn't really give any thought to the layout on this one. You proposed changes are already a lot better, so I'll go ahead and implement them.

I suspect this page deserves further design review and iteration, but at least it won't be completely shit. :-D

Btw, on the H3 vs H2 size - I'll use Gravity's text size utility classes to do this now, but we might want to consider reviewing the default sizes we assign to different heading levels in Gravity too.